senders/discord/init.go

```diff
@@ -11,16 +11,17 @@ import (
 )
 
 const (
-	messenger       = "discord"
-	discordLockName = "moira-discord-users:moira-bot-host"
-	discordLockTTL  = 30 * time.Second
-	workerName      = "DiscordBot"
+	messenger      = "discord"
+	discordLockKey = "moira-discord-users:moira-bot-host:"
```

Comment:
Мб так? Или у нас есть какая-то конвенция именования таких префиксов?

```suggestion
	discordLockPrefix = "moira-discord-users:moira-bot-host:"
```

Reply:
Поменял